pages/lists/[listId]/overview.tsx

```diff
@@ -12,10 +14,50 @@ import { getAvatarByUsername } from "lib/utils/github";
 import useListContributions from "lib/hooks/useListContributions";
 import useListStats from "lib/hooks/useListStats";
 import HighlightCard from "components/molecules/HighlightCard/highlight-card";
+import { fetchApiData, validateListPath } from "helpers/fetchApiData";
+import ClientOnly from "components/atoms/ClientOnly/client-only";
 
 type ContributorPrMap = { [contributor: string]: DbRepoPR };
 
-const ListsOverview = (): JSX.Element => {
+interface ListsOverviewProps {
+  list: DBList;
+  numberOfContributors: number;
+}
+
+export const getServerSideProps = async (ctx: GetServerSidePropsContext) => {
+  const supabase = createPagesServerClient(ctx);
+
+  const {
+    data: { session },
+  } = await supabase.auth.getSession();
+  const bearerToken = session ? session.access_token : "";
+
+  const { listId } = ctx.params as { listId: string };
+  const limit = 10; // Can pull this from the querystring in the future
+  const [{ data, error: contributorListError }, { data: list, error }] = await Promise.all([
+    fetchApiData<PagedData<DBListContributor>>({
+      path: `lists/${listId}/contributors?limit=${limit}`,
+      bearerToken,
+      pathValidator: validateListPath,
+    }),
+    fetchApiData<DBList>({ path: `lists/${listId}`, bearerToken, pathValidator: validateListPath }),
+  ]);
+
+  if (error?.status === 404) {
+    return {
+      notFound: true,
+    };
+  }
+
+  return {
+    props: {
+      list,
+      numberOfContributors: data?.meta.itemCount || 0,
+    },
+  };
+};
+
+const ListsOverview = ({ list, numberOfContributors }: ListsOverviewProps): JSX.Element => {
```

Comment:
@brandonroberts, I've refactored the Overview page so it works with the way the layout currently loads for the Contributors List.